invenio_opendata/base/templates/data/news_db.html

```diff
@@ -1,5 +1,19 @@
 {% set news = [
   (
+  'Explore ATLAS Open Data resources', '
+  '<p>
+    Starting today, the data from 100 trillion proton collisions is now public on the Open Data Portal)! This marks the world's first open release of 8 TeV data, gathered from the Large Hadron Collider in 2012.
+  </p>
+  <p>
+    Unique to the ATLAS Open Data release is its strong emphasis on learning. Hence, additional resources are available via the ATLAS educational platform which guides users through the data, explains how to visualize and how to download and use the data, and even provides open-source software. As part of this, ATLAS has made seven physics analyses available for users to get started with their research. By releasing additional resources and comprehensive documentation ATLAS aims at bridging the gap between phycisists and the public.
```

Comment:
Is it really unique to ATLAS? :P 

Anyway, spelling should be -ise, not -ize. So, "visualise".